docs/how_to/install.rst

```diff
@@ -1,671 +1,519 @@
-the django CMS `extensions page`_.
-
-.. _snippet_plugin: https://github.com/divio/djangocms-snippet
-.. _extensions page: http://www.django-cms.org/en/extensions/
-
-In addition, make sure you uncomment (enable) ``'django.contrib.admin'``
-
-You may also wish to use `django-filer`_ and its components with the `django
-CMS plugin`_ instead of the :mod:`djangocms_file`, :mod:`djangocms_picture`,
-:mod:`djangocms_teaser` and :mod:`djangocms_video` core plugins. In this case
-you should check the `django-filer documentation
-<django-filer:installation_and_configuration>`_ and `django CMS plugin documentation`_
-for detailed installation information, and then return to this tutorial.
-
-.. _django-filer: https://github.com/stefanfoulis/django-filer
-.. _django CMS plugin: https://github.com/stefanfoulis/cmsplugin-filer
-.. _django CMS plugin documentation: https://github.com/stefanfoulis/cmsplugin-filer#installation
-
-If you opt for the core plugins you should take care that directory to which
-the :setting:`CMS_PAGE_MEDIA_PATH` setting points (by default ``cms_page_media/``
-relative to :setting:`django:MEDIA_ROOT`) is writeable by the user under which Django
-will be running. If you have opted for django-filer there is a similar requirement
-for its configuration.
-
-You need to add the django CMS middlewares to your :setting:`django:MIDDLEWARE_CLASSES`
-at the right position::
-
-    MIDDLEWARE_CLASSES = (
-        'cms.middleware.utils.ApphookReloadMiddleware',
-        'django.contrib.sessions.middleware.SessionMiddleware',
-        'django.middleware.csrf.CsrfViewMiddleware',
-        'django.contrib.auth.middleware.AuthenticationMiddleware',
-        'django.contrib.messages.middleware.MessageMiddleware',
-        'django.middleware.locale.LocaleMiddleware',
-        'django.middleware.common.CommonMiddleware',
-        'cms.middleware.user.CurrentUserMiddleware',
-        'cms.middleware.page.CurrentPageMiddleware',
-        'cms.middleware.toolbar.ToolbarMiddleware',
-        'cms.middleware.language.LanguageCookieMiddleware',
-    )
+This is worth explaining in a little detail:
 
-Notice that django CMS v3.2 introduces a new middleware:
-``cms.middleware.utils.ApphookReloadMiddleware``. This should be placed very
-near the top of your middleware classes tuple/list.
+* ``{% load cms_tags sekizai_tags %}`` loads the template tag libraries we use in the template.
+* ``{% page_attribute "page_title" %}`` extracts the page's ``page_title`` attribute.
+* ``{% render_block "css" %}`` and ``{% render_block "js" %}`` are template tags that load blocks of HTML defined by
+  Django applications. django CMS defines blocks for CSS and JavaScript, and requires these two Sekizai tags. We
+  recommended placing ``{% render_block "css" %}`` just before the ``</head>`` tag, and and ``{% render_block "js" %}``
+  tag just before the ``</body>``.
+* ``{% cms_toolbar %}`` renders the django CMS toolbar.
```

Comment:
should we point out here that this tag cannot be used inside another tag?
or is it somewhere else?